backend_plugin/src/main/evaluators/BlockCommentEvaluator.java

```diff
@@ -15,6 +15,11 @@
 	private IRegion prevRegion;
 	private int prevOffset;
 	private String prevInsert;
+	private int currentOffset;
```

Comment:
Didn't catch this last time, but having a comment to explain what `currentOffset` and `currentInsert` are would be useful.